README.md

```diff
@@ -16,20 +16,23 @@ def application do
   [applications: [:phoenix_pubsub]]
 end
 ```
-        
+
 ## Initialization (without Phoenix)
 ```elixir
 defmodule MyApp do
   use Application
 
   def start(_type, _args) do
-    import Supervisor.Spec, warn: false
 
     children = [
-      supervisor(Phoenix.PubSub.PG2, [MyApp.PubSub, []])
+      { Phoenix.PubSub.PG2, [ MyApp.PubSub, [] ]}
```

Comment:
I believe the preferred style would be without spaces around `{}[]`

Reply:
Might well be, but I don't use the formatter on religious grounds :)

Reply:
This project doesn't use the formatter. Even though, removing the spaces would make it consistent with the remaining code in the repository.